packages/components/src/components/modal/_modal.scss

```diff
@@ -135,6 +135,14 @@
     margin-bottom: $carbon--spacing-09;
     color: $text-01;
     font-weight: 400;
+
+    &:focus {
+      // @question: couldn't find a helper that this the wider focus style
```

Comment:
I believe the `focus-outline('outline')` mixin can be used here

Reply:
That did the trick.  Thanks @emyarod!